lib/internal/http2/core.js

```diff
@@ -442,7 +445,7 @@ function setupHandle(session, socket, type, options, settings) {
     handle.onstreamclose = onSessionStreamClose;
     handle.onerror = onSessionError;
     handle.onread = onSessionRead;
-    handle.ongetpadding = onSelectPadding;
+    handle.ongetpadding = onSelectPadding(options.selectPadding);
```

Comment:
yeah, good point. if `options.selectPadding` is `undefined`, `handle.ongetpadding` should be `undefined`, in which case no padding would be applied.